traefik_kop.go

```diff
@@ -101,6 +101,11 @@ func replaceIPs(dockerClient client.APIClient, conf *dynamic.Configuration, ip s
 		for svcName, svc := range conf.HTTP.Services {
 			logrus.Debugf("found http service: %s", svcName)
 			for i := range svc.LoadBalancer.Servers {
+				network_ip := "" // fix. persists between loops?
+				network_ip = getContainerIP(dockerClient, conf, "http", svcName)
```

Comment:
It shouldn't persist. Not sure what issue you saw, but you should be able to remove line 104 and simply initialize the variable here with `network_ip := getContainerIP(dockerClient, conf, "http", svcName)`